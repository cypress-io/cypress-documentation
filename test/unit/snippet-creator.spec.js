require('../spec_helper')

const snippetCreator = require('../../cy_scripts/snippet-creator/cli')

describe('Snippet Creator', () => {
  // it('can be launched as a process', () => {
  //   exec('node ../../cy_scripts/snippet-creator aaaa')
  //   // .then(unit test it)
  // })

  it('errors out if can\'t find input file', () => {
    return snippetCreator.start(['node', 'snip', 'asdf'])
    .then(snapshot)
  })

  it('errors if no argument provided', () => {
    return snippetCreator.start(['node', 'snip'])
    .then(snapshot)
  })
})
