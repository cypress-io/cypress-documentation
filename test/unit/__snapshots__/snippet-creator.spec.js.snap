// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snippet Creator errors if no argument provided 1`] = `"[Snippet Creator] Error: Script takes exactly 1 argument, received 0"`;

exports[`Snippet Creator errors out if can't find input file 1`] = `"[Snippet Creator] Error: Cannot find input file: asdf"`;
